setup.cfg

```diff
@@ -45,19 +45,6 @@ ignore = E402,  # module level import not at top of file
          E703,  # statement ends with a semicolon
 
 exclude =
-    doc/source/whatsnew/v0.7.0.rst
-    doc/source/whatsnew/v0.7.3.rst
-    doc/source/whatsnew/v0.8.0.rst
-    doc/source/whatsnew/v0.9.0.rst
-    doc/source/whatsnew/v0.9.1.rst
-    doc/source/whatsnew/v0.10.0.rst
-    doc/source/whatsnew/v0.10.1.rst
-    doc/source/whatsnew/v0.11.0.rst
-    doc/source/whatsnew/v0.12.0.rst
-    doc/source/whatsnew/v0.13.0.rst
-    doc/source/whatsnew/v0.13.1.rst
-    doc/source/whatsnew/v0.14.0.rst
-    doc/source/whatsnew/v0.14.1.rst
```

Comment:
I'm a bit confused about this. Are all the others correct and we forgot to remove them? I'd expect just the `v0.13.*` to be removed

Reply:
Sorry if I am butting in. Issues related to v0.14* was fixed and merged. I think a merge with upstream/master will resolve this.

Reply:
Thanks for the feedback on this, I hadn't had to resolve a merge conflict before and I was doing it wrong.  All sorted now I believe.